bin/cloud-provision.sh

```diff
@@ -57,7 +58,10 @@ send_and_execute(){
 }
 
 copy_credentials() {
-    scp -r $OPENSTACK_CREDENTIALS_PATH ubuntu@$INSTANCE_IP:$JENKINS_HOME
+    # first novarc file in $OPENSTACK_CREDENTIALS_PATH
+    first_novarc=$(ls $OPENSTACK_CREDENTIALS_PATH/novarc* | sort -d | head -1)
```

Comment:
You are right, if there are multiple novarc files there's no way to select one of them.

Because we don't need anymore the nova keys we could pass the novarc path instead of the credentials path as a parameter to the provision script, something like:

    ./bin/cloud-provision.sh ~/.canonistack/novarc_lcy02

What do you think?